tests/e2e/utils/workspace/IUrlResolver.ts

```diff
@@ -0,0 +1,13 @@
+/*********************************************************************
+ * Copyright (c) 2019 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+export interface IApiUrlResolver {
+    getWorkspaceApiUrl(namespace: string, workspacename: string): string;
```

Comment:
There are more things implemented in tests/e2e/utils/workspace/ApiUrlResolver.ts:
1. Method `getWorkspacesApiUrl(namespace: string)`
2. Property `dashboardApiUrl: string = 'dashboard/api/namespace';`

What do you think about declaring them in the interface?.

Reply:
What will you have downstream then?
Something like the following? 
```
@injectable()
export class ApiUrlResolver implements IApiUrlResolver {
    private  dashboardApiUrl: string = 'dashboard/api/namespace';

    public getWorkspaceApiUrl(namespace: string, workspaceName: string) {
        return `${this.getWorkspacesApiUrl(namespace)}/${workspaceName}`;
    }

    public getWorkspacesApiUrl(namespace: string) {
        return `${this.dashboardApiUrl}/${namespace}/devworkspaces`;
    }
}
```

Reply:
Yes

Reply:
Change the interface according to suggestion